### DISCUSSION TITLE
Osu Crashing my other games

### DISCUSSION THREAD

**Discussion by UniversalBlood** [ppy/osu#20529] [Q&A]
Does any one know how Osu! Lazer would be crashing my other games when they are loaded up? it never did that before

**Comment by peppy** [comment]
Usually NVIDIA overlays or otherwise. Can you provide logs from event viewer showing the crashes?

**Reply by UniversalBlood** [reply]
Uhhh to be honest i dont really know how to show the crash logs

**Reply by ghost** [reply]
#15188
